rules/falco_rules.yaml

```diff
@@ -3147,13 +3150,14 @@
   enabled: false
   tags: [host, container, network, mitre_command_and_control, TA0011]
 
-- list: white_listed_modules
```

Comment:
who do we remove the whitelist ?

Reply:
For me now given that the rule is changing, it makes sense having a whitelist on *container images* that can load modules

Reply:
did you add it in the condition

Reply:
no, that's what I was asking (see above), let's wait for Melissa :)

Reply:
with the whitelisted images (e.g. falco), I think we're good to go.

Reply:
Added, this is great feedback, ty :)